tests/Notifications/NotificationSenderTest.php

```diff
@@ -138,7 +161,7 @@ public function via($notifiable)
     }
 }
 
-class DummyNotificationWithDatabaseVia extends Notification
+class DummyNotificationWithDatabaseVia extends Notification implements ShouldQueue
```

Comment:
You can't change existing tests like this.

Reply:
Oups, fail during my tests.